sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/microsoft/opentelemetry/exporter/azuremonitor/export/__init__.py

```diff
@@ -35,8 +34,8 @@ class BaseExporter:
         options: :doc:`export.options` to allow configuration for the exporter
     """
 
-    def __init__(self, **options):
```

Comment:
We have `**options` in ``AzureMonitorTraceExporter` but `**kwargs` in `BaseExporter`?